download/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-<meta http-equiv="refresh" content="0; URL='../download.html'" />
+<meta http-equiv="refresh" content="0; URL='https://forum.puppylinux.com/puppy-linux-collection'" />
```

Comment:
Yes, and do the same thing in download.html, then empty that page.

Reply:
I don't think there is a download.html ??

Reply:
Both the current .htm redirects give a "404 Not Found" error:

e.g.
https://puppylinux.com/faq
https://puppylinux.com/forums

as does:
https://puppylinux.com/download
which doesn't have a download.htm redirect

https://puppylinux-woof-ce.github.io/download
does work

Reply:
Why is the index page an .html page and not .md? Maybe download.md can be replaced with an empty HTML page with that redirect?

Reply:
faq.htm
forums.htm

are like that and don't work.............from the puppylinux.com domain